### ISSUE TITLE
Couldn't compile a test in MacOS

### ISSUE THREAD

**Issue by wangtsiao** [privacy-scaling-explorations/zkevm-circuits#1410]
### What command(s) is the bug in?

make test

### Describe the bug

Here is part of the error message.

```
.....
  = note: Undefined symbols for architecture x86_64:
            "_SecCertificateCopyData", referenced from:
                _crypto/x509/internal/macos.x509_SecCertificateCopyData_trampoline.abi0 in libgeth_utils-f82ba25d13d148dc.rlib(go.o)
            "_SecCertificateCreateWithData", referenced from:
                _crypto/x509/internal/macos.x509_SecCertificateCreateWithData_trampoline.abi0 in libgeth_utils-f82ba25d13d148dc.rlib(go.o)
            "_SecPolicyCreateSSL", referenced from:
                _crypto/x509/internal/macos.x509_SecPolicyCreateSSL_trampoline.abi0 in libgeth_utils-f82ba25d13d148dc.rlib(go.o)
            "_SecTrustCreateWithCertificates", referenced from:
                _crypto/x509/internal/macos.x509_SecTrustCreateWithCertificates_trampoline.abi0 in libgeth_utils-f82ba25d13d148dc.rlib(go.o)
            "_SecTrustEvaluateWithError", referenced from:
                _crypto/x509/internal/macos.x509_SecTrustEvaluateWithError_trampoline.abi0 in libgeth_utils-f82ba25d13d148dc.rlib(go.o)
            "_SecTrustGetCertificateAtIndex", referenced from:
                _crypto/x509/internal/macos.x509_SecTrustGetCertificateAtIndex_trampoline.abi0 in libgeth_utils-f82ba25d13d148dc.rlib(go.o)
            "_SecTrustGetCertificateCount", referenced from:
                _crypto/x509/internal/macos.x509_SecTrustGetCertificateCount_trampoline.abi0 in libgeth_utils-f82ba25d13d148dc.rlib(go.o)
            "_SecTrustSetVerifyDate", referenced from:
                _crypto/x509/internal/macos.x509_SecTrustSetVerifyDate_trampoline.abi0 in libgeth_utils-f82ba25d13d148dc.rlib(go.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: aborting due to previous error

Couldn't compile the test.

failures:
    src/test_ctx.rs - test_ctx::TestContext (line 27)

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 8.85s

error: doctest failed, to rerun pass `-p mock --doc`
make: *** [test] Error 101
````

**Comment by ChihChengLiang** [comment]
It seemed to fail the doc test. I can reproduce and narrow the failure to `cargo test --release -p mock --all-features --doc`

**Comment by wangtsiao** [comment]
The issue was resolved by [PR#1412](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1412/files), thank you.
